avocado/plugins/runner_nrunner.py

```diff
@@ -48,11 +49,12 @@ def initialize(self):
                                  help_msg=help_msg,
                                  key_type=bool)
 
+        port = find_free_port()
         help_msg = ('URI for listing the status server. Usually '
                     'a "HOST:PORT" string')
         settings.register_option(section=section,
                                  key='status_server_listen',
-                                 default='127.0.0.1:8888',
+                                 default='127.0.0.1:%u' % port,
```

Comment:
This is something that it was discussed before and we have agreed that having a default random port seems really strange. What I'm missing here?

Reply:
Oh, this is workaround, and marked as `[REMOVE ME]`.  The referenced issue is expected to solve this properly.